viewing_party/party.py

```diff
@@ -0,0 +1,182 @@
+
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        new_movie_dict = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+            }
+        return new_movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title): 
+    watch_list = user_data["watchlist"] 
+    watched = user_data["watched"]
+
+    
+    for movie in watch_list:
+        if title in movie["title"]:
+            watch_list.remove(movie)
+            watched.append(movie)
+    return user_data#def create_movie(title,genre,rating):
+    #if title and genre and rating:
+#         movie_dict = {"title": title,
+#         "genre" :genre,
+#         "rating": rating,
+#         }
+#         return movie_dict
+#     else:
+#         return None
+
+
+# def add_to_watched(user_data, movie):
+#     watched = user_data["watched"]
+#     if movie:
+#         watched.append(movie)
+#     else:
+#         watched = []
+#     return user_data
```

Comment:
When you are finished with a project/branch you would remove unused code when making a pull request